api/BUILD

```diff
@@ -209,6 +209,7 @@ proto_library(
         "//envoy/extensions/filters/http/ratelimit/v3:pkg",
         "//envoy/extensions/filters/http/rbac/v3:pkg",
         "//envoy/extensions/filters/http/router/v3:pkg",
+        "//envoy/extensions/filters/http/set_metadata/v3:pkg",
```

Comment:
Need to update https://github.com/envoyproxy/envoy/blob/main/CODEOWNERS

Reply:
Yes, Snow and you.

Reply:
Sorry missed this answer. Done in d139579d9ed5c2a00090f4e75f61df26d0d8eda9 !